### PR TITLE
Script to download ChEMBL SMILES entries and convert them to h5 input…

### PR DESCRIPTION
**Pull request by dakoner**
… for preprocessing.

### CODE REVIEW

File: import_chembl.py [hunk 1]
```diff
@@ -0,0 +1,35 @@
+import argparse
+import gzip
+import pandas
+import tempfile
+import urllib
+
+def get_arguments():
+    parser = argparse.ArgumentParser(description='Download ChEMBL entries and convert them to input for preprocessing')
+    DEFAULT_URI = 'ftp://ftp.ebi.ac.uk/pub/databases/chembl/ChEMBLdb/latest/chembl_22_chemreps.txt.gz'
```
Comment (maxhodak) [inline import_chembl.py]: please pull this out to between the imports and the `def get_arguments` declaration following the example in the other scripts

**Review by maxhodak — CHANGES_REQUESTED** [review]
`download_chembl.py` would be more accurate than `import_chembl.py` otherwise looks pretty good

**Comment by dakoner** [comment]
Renamed file,
factored DEFAULT_URI to top scope.
(I did a git commit --amend, hoping that github updated the pull request to include those changed, please check that you see these fixes.)

**Comment by maxhodak** [comment]
it's good practice to pull out all defaults (file name: 'chembl_smiles.h5') but i'll merge this for now since i missed that on my earlier review and it doesn't break anything

**Pull request merged**